test/runtests.jl

```diff
@@ -636,3 +636,17 @@ end
 
     @test beta(big(1.0),big(1.2)) ≈ beta(1.0,1.2) rtol=4*eps()
 end
+
+@testset "lbinomial" begin
+    lbfixed(n) = k -> lbinomial(n, k)
```

Comment:
Why is this necessary? You can write `lbfixed(200).(0:200)` below as `lbinomial.(200, 0:200)`.

Reply:
Note that the same applies to the function defined directly below this line.